plugins/modules/source_control/gitlab/gitlab_group_members.py

```diff
@@ -0,0 +1,232 @@
+
+        except (gitlab.exceptions.GitlabDeleteError) as e:
+            self._module.fail_json(
+                msg="Failed to remove member from GitLab group, ID %s: %s" % (gitlab_group_id, e))
+
+
+def main():
+    module = AnsibleModule(
+        argument_spec=dict(
+            server_url=dict(type='str', required=True),
+            access_token=dict(type='str', required=True, no_log=True),
+            gitlab_group=dict(type='str', required=True),
+            gitlab_user=dict(type='str', required=True),
+            state=dict(type='str', default='present', choices=['present', 'absent']),
+            access_level=dict(type='str', required=False, choices=['guest', 'reporter', 'developer', 'maintainer', 'owner'])
+        ),
+        supports_check_mode=True
+    )
+
+    if not HAS_PY_GITLAB:
+        module.fail_json(msg=missing_required_lib('python-gitlab', url='https://python-gitlab.readthedocs.io/en/stable/'), exception=GITLAB_IMP_ERR)
+
+    server_url = module.params['server_url']
+    access_token = module.params['access_token']
+    gitlab_group = module.params['gitlab_group']
+    gitlab_user = module.params['gitlab_user']
+    state = module.params['state']
+    access_level = module.params['access_level']
+
+    # convert access level string input to int
+    if access_level:
+        access_level_int = {
+            'guest': gitlab.GUEST_ACCESS,
+            'reporter': gitlab.REPORTER_ACCESS,
+            'developer': gitlab.DEVELOPER_ACCESS,
+            'maintainer': gitlab.MAINTAINER_ACCESS,
+            'owner': gitlab.OWNER_ACCESS
+        }
+
+        access_level = access_level_int[access_level]
+
+    # connect to gitlab server
+    try:
+        gl = gitlab.Gitlab(server_url, private_token=access_token)
+    except (gitlab.exceptions.GitlabAuthenticationError, gitlab.exceptions.GitlabGetError) as e:
+        module.fail_json(msg="Failed to authenticate to GitLab Server: %s: %s" % (server_url, e))
+
+    group = GitLabGroup(module, gl)
+
+    user_exists = group.does_user_exist(gitlab_user)
+    group_exists = group.does_group_exist(gitlab_group)
+
+    # group doesn't exist
+    if not group_exists:
+        module.fail_json(msg="group '%s' not found." % gitlab_group)
+
+    # user doesn't exist
+    if not user_exists:
+        module.fail_json(msg="user '%s' not found." % gitlab_user)
+
+    if group_exists:
+        # get group id
+        gitlab_group_id = group_exists.id
+
+        if user_exists:
+            # get user id
+            gitlab_user_id = user_exists.id
+            is_user_a_member = group.is_user_a_member(gitlab_group_id, gitlab_user_id)
+
+            # check if the user is a member in the group
+            if not is_user_a_member:
+                if state == 'present':
+                    if not access_level:
+                        module.fail_json(msg="Access level must be entered for adding member to GitLab group '%s'." % gitlab_user)
```

Comment:
You insert the user's name as the group name. Same for the next ones.

Reply:
@felixfontein , can you elaborate on this? 

Fail msg will print the user name who you need to specify the access level for.

Reply:
Yes, but the way you formulated the sentence, it says in English that the group the member is added to is called (the value of) `gitlab_user`.

Reply:
okay. It should be all set now. 
I went over all the msgs in the module and moved the parameter next to what it points to.